PrivateCollections/Germany/Neukirchen003.xml

```diff
@@ -0,0 +1,1145 @@
+                      </msItem>
+                      <msItem xml:id="ms_i1.1.149">
+                        <locus from="105v" to="106r"></locus>
+                        <title type="complete" ref="LIT2000Mazmur#Ps149"></title>
+                        <textLang mainLang="gez"></textLang>
+                      </msItem>
+                      <msItem xml:id="ms_i1.1.150">
+                        <locus from="106r" to="106v"></locus>
+                        <title type="complete" ref="LIT2000Mazmur#Ps150"></title>
+                        <textLang mainLang="gez"></textLang>
+                      </msItem>
+                      <msItem xml:id="ms_i1.1.151">
+                        <locus target="#106v"></locus>
+                        <title type="complete" ref="LIT2000Mazmur#Ps151"></title>
+                        <textLang mainLang="gez"></textLang>
+                      </msItem>
+                    </msItem>
+                    <msItem xml:id="ms_i1.2">
+                      <locus from="107r" to="117v"></locus>
+                      <title type="complete" ref="LIT1828Mahale"></title>
+                        <textLang mainLang="gez"></textLang>
+                    </msItem>
+                    <msItem xml:id="ms_i1.3">
+                      <locus from="118r" to="124v"></locus>
+                      <title type="complete" ref="LIT2362Songof"></title>
+                      <textLang mainLang="gez"></textLang>
+                    </msItem>
+                    <msItem xml:id="ms_i1.4">
+                      <locus from="125r" to="137v"></locus>
+                      <title type="complete" ref="LIT2509Weddas"></title>
+                      <textLang mainLang="gez"></textLang>
+                    </msItem>
+                  </msItem>
+
+               </msContents>
+               <physDesc>
+                  <objectDesc form="Codex">
+                     <supportDesc>
+                        <support>
+
+                           <material key="parchment"/>
+                        </support>
+                        <extent>
+                           <measure unit="leaf">138</measure>
+                        <measure unit="quire">13</measure>
+                           <dimensions type="outer" unit="mm">
+                              <height>193</height>
+                              <width>118</width>
+                           </dimensions>
+                           <dimensions type="leaf" unit="mm" xml:id="leafdim">
+                             <height>193</height>
+                             <width>116</width>
+                           </dimensions>
+                           <note corresp="#leafdim">
+                             Data on leaves dimensions
+                             taken from <locus target="#27r"></locus>.
+                           </note>
+                        </extent>
+                        <foliation>
+                          <list>
```

Comment:
If this is the only foliation present in the manuscript, having a list seems a bit redundant (but not wrong either).

Reply:
There is only one. I just wanted to make clear, that the foliation was done by me

Reply:
The text is completely fine and clear, you could have it just inside `<foliation>` without any further elements.

Reply:
I changed this, is it ok like that?